FirebaseAuth/Tests/Sample/Podfile

```diff
@@ -4,15 +4,15 @@
 
 source 'https://github.com/firebase/SpecsDev.git'
 source 'https://github.com/firebase/SpecsStaging.git'
-source 'https://cdn.cocoapods.org/'
+source "https://github.com/CocoaPods/Specs.git"
 
 use_frameworks!
 
 target 'AuthSample' do
-  platform :ios, '11.0'
+  platform :ios, '12.0'
   pod 'FirebaseCore', :path => '../../../'
   pod 'FirebaseCoreInternal', :path => '../../../'
-  pod 'FirebaseAuth', :path => '../../../', :testspecs => ['unit']
```

Comment:
This works if you set the environment variable expected by the podspec first.

Reply:
Yeah, thanks for pointing it out.